perfact/zodbsync/zodbsync.py

```diff
@@ -138,7 +140,9 @@ def mod_write(data, parent=None, obj_id=None, override=False, root=None,
     d = dict(data)
     meta_type = d['type']
 
-    if default_owner is not None and 'owner' not in d:
+    no_owner_given = 'owner' not in d
+
+    if (default_owner and no_owner_given) or (force_default_owner):
```

Comment:
I guess giving no `default_owner` but setting `force_default_owner` is not supported and does not need to give a sensible result.